[2018-04-23 10:50:42] (ERROR @ /mnt/sdb1/quo-group/core/model/modx/modx.class.php : 697) PHP warning: Invalid argument supplied for foreach()
[2018-04-23 10:50:42] (ERROR @ /mnt/sdb1/quo-group/core/model/modx/modx.class.php : 697) PHP warning: Invalid argument supplied for foreach()
[2018-04-23 10:53:02] (ERROR @ /mnt/sdb1/quo-group/core/model/modx/modx.class.php : 697) PHP warning: Invalid argument supplied for foreach()
[2018-04-23 10:53:02] (ERROR @ /mnt/sdb1/quo-group/core/model/modx/modx.class.php : 697) PHP warning: Invalid argument supplied for foreach()
[2018-04-23 11:04:06] (ERROR @ /mnt/sdb1/quo-group/core/model/modx/transport/modtransportpackage.class.php : 384) Невозможно открыть файл для чтения: http://modx.s3.amazonaws.com/extras/4d556c62b2b083396d000b9c/formit-4.0.0-pl.transport.zip?revolution_version=Revolution-2.6.3-pl
[2018-04-23 11:33:56] (ERROR @ /mnt/sdb1/quo-group/core/model/modx/modcachemanager.class.php : 394) Error caching action map mgr/actions
[2018-04-23 11:38:59] (ERROR @ /mnt/sdb1/quo-group/core/model/modx/modcachemanager.class.php : 394) Error caching action map mgr/actions
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:53] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modExtensionPackage`.`id`, `modExtensionPackage`.`namespace`, `modExtensionPackage`.`name`, `modExtensionPackage`.`path`, `modExtensionPackage`.`table_prefix`, `modExtensionPackage`.`service_class`, `modExtensionPackage`.`service_name`, `modExtensionPackage`.`created_at`, `modExtensionPackage`.`updated_at`, Namespace.path AS namespace_path FROM `ewal_extension_packages` AS `modExtensionPackage` JOIN `ewal_namespaces` `Namespace` ON `modExtensionPackage`.`namespace` =  `Namespace`.`name` ORDER BY namespace ASC 
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:53] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modSession`.`id` AS `modSession_id`, `modSession`.`access` AS `modSession_access`, `modSession`.`data` AS `modSession_data` FROM `ewal_session` AS `modSession` WHERE `modSession`.`id` = ? 
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:53] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 263) Error preparing statement for query: SELECT `modSession`.`id` AS `modSession_id`, `modSession`.`access` AS `modSession_access`, `modSession`.`data` AS `modSession_data` FROM `ewal_session` AS `modSession` WHERE `modSession`.`id` = ?  - 
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:53] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modChunk`.`id` AS `modChunk_id`, `modChunk`.`source` AS `modChunk_source`, `modChunk`.`property_preprocess` AS `modChunk_property_preprocess`, `modChunk`.`name` AS `modChunk_name`, `modChunk`.`description` AS `modChunk_description`, `modChunk`.`editor_type` AS `modChunk_editor_type`, `modChunk`.`category` AS `modChunk_category`, `modChunk`.`cache_type` AS `modChunk_cache_type`, `modChunk`.`snippet` AS `modChunk_snippet`, `modChunk`.`locked` AS `modChunk_locked`, `modChunk`.`properties` AS `modChunk_properties`, `modChunk`.`static` AS `modChunk_static`, `modChunk`.`static_file` AS `modChunk_static_file`, `Source`.`id` AS `Source_id`, `Source`.`name` AS `Source_name`, `Source`.`description` AS `Source_description`, `Source`.`class_key` AS `Source_class_key`, `Source`.`properties` AS `Source_properties`, `Source`.`is_stream` AS `Source_is_stream` FROM `ewal_site_htmlsnippets` AS `modChunk` LEFT JOIN `ewal_media_sources` `Source` ON `modChunk`.`source` = `Source`.`id` WHERE `modChunk`.`name` = ? ORDER BY `modChunk`.`id` ASC 
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:53] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 546) Error  preparing statement: SELECT `modChunk`.`id` AS `modChunk_id`, `modChunk`.`source` AS `modChunk_source`, `modChunk`.`property_preprocess` AS `modChunk_property_preprocess`, `modChunk`.`name` AS `modChunk_name`, `modChunk`.`description` AS `modChunk_description`, `modChunk`.`editor_type` AS `modChunk_editor_type`, `modChunk`.`category` AS `modChunk_category`, `modChunk`.`cache_type` AS `modChunk_cache_type`, `modChunk`.`snippet` AS `modChunk_snippet`, `modChunk`.`locked` AS `modChunk_locked`, `modChunk`.`properties` AS `modChunk_properties`, `modChunk`.`static` AS `modChunk_static`, `modChunk`.`static_file` AS `modChunk_static_file`, `Source`.`id` AS `Source_id`, `Source`.`name` AS `Source_name`, `Source`.`description` AS `Source_description`, `Source`.`class_key` AS `Source_class_key`, `Source`.`properties` AS `Source_properties`, `Source`.`is_stream` AS `Source_is_stream` FROM `ewal_site_htmlsnippets` AS `modChunk` LEFT JOIN `ewal_media_sources` `Source` ON `modChunk`.`source` = `Source`.`id` WHERE `modChunk`.`name` = ? ORDER BY `modChunk`.`id` ASC  - 
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:53] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modChunk`.`id` AS `modChunk_id`, `modChunk`.`source` AS `modChunk_source`, `modChunk`.`property_preprocess` AS `modChunk_property_preprocess`, `modChunk`.`name` AS `modChunk_name`, `modChunk`.`description` AS `modChunk_description`, `modChunk`.`editor_type` AS `modChunk_editor_type`, `modChunk`.`category` AS `modChunk_category`, `modChunk`.`cache_type` AS `modChunk_cache_type`, `modChunk`.`snippet` AS `modChunk_snippet`, `modChunk`.`locked` AS `modChunk_locked`, `modChunk`.`properties` AS `modChunk_properties`, `modChunk`.`static` AS `modChunk_static`, `modChunk`.`static_file` AS `modChunk_static_file`, `Source`.`id` AS `Source_id`, `Source`.`name` AS `Source_name`, `Source`.`description` AS `Source_description`, `Source`.`class_key` AS `Source_class_key`, `Source`.`properties` AS `Source_properties`, `Source`.`is_stream` AS `Source_is_stream` FROM `ewal_site_htmlsnippets` AS `modChunk` LEFT JOIN `ewal_media_sources` `Source` ON `modChunk`.`source` = `Source`.`id` WHERE `modChunk`.`name` = ? ORDER BY `modChunk`.`id` ASC 
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:53] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 546) Error  preparing statement: SELECT `modChunk`.`id` AS `modChunk_id`, `modChunk`.`source` AS `modChunk_source`, `modChunk`.`property_preprocess` AS `modChunk_property_preprocess`, `modChunk`.`name` AS `modChunk_name`, `modChunk`.`description` AS `modChunk_description`, `modChunk`.`editor_type` AS `modChunk_editor_type`, `modChunk`.`category` AS `modChunk_category`, `modChunk`.`cache_type` AS `modChunk_cache_type`, `modChunk`.`snippet` AS `modChunk_snippet`, `modChunk`.`locked` AS `modChunk_locked`, `modChunk`.`properties` AS `modChunk_properties`, `modChunk`.`static` AS `modChunk_static`, `modChunk`.`static_file` AS `modChunk_static_file`, `Source`.`id` AS `Source_id`, `Source`.`name` AS `Source_name`, `Source`.`description` AS `Source_description`, `Source`.`class_key` AS `Source_class_key`, `Source`.`properties` AS `Source_properties`, `Source`.`is_stream` AS `Source_is_stream` FROM `ewal_site_htmlsnippets` AS `modChunk` LEFT JOIN `ewal_media_sources` `Source` ON `modChunk`.`source` = `Source`.`id` WHERE `modChunk`.`name` = ? ORDER BY `modChunk`.`id` ASC  - 
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:53] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modChunk`.`id` AS `modChunk_id`, `modChunk`.`source` AS `modChunk_source`, `modChunk`.`property_preprocess` AS `modChunk_property_preprocess`, `modChunk`.`name` AS `modChunk_name`, `modChunk`.`description` AS `modChunk_description`, `modChunk`.`editor_type` AS `modChunk_editor_type`, `modChunk`.`category` AS `modChunk_category`, `modChunk`.`cache_type` AS `modChunk_cache_type`, `modChunk`.`snippet` AS `modChunk_snippet`, `modChunk`.`locked` AS `modChunk_locked`, `modChunk`.`properties` AS `modChunk_properties`, `modChunk`.`static` AS `modChunk_static`, `modChunk`.`static_file` AS `modChunk_static_file`, `Source`.`id` AS `Source_id`, `Source`.`name` AS `Source_name`, `Source`.`description` AS `Source_description`, `Source`.`class_key` AS `Source_class_key`, `Source`.`properties` AS `Source_properties`, `Source`.`is_stream` AS `Source_is_stream` FROM `ewal_site_htmlsnippets` AS `modChunk` LEFT JOIN `ewal_media_sources` `Source` ON `modChunk`.`source` = `Source`.`id` WHERE `modChunk`.`name` = ? ORDER BY `modChunk`.`id` ASC 
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:53] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 546) Error  preparing statement: SELECT `modChunk`.`id` AS `modChunk_id`, `modChunk`.`source` AS `modChunk_source`, `modChunk`.`property_preprocess` AS `modChunk_property_preprocess`, `modChunk`.`name` AS `modChunk_name`, `modChunk`.`description` AS `modChunk_description`, `modChunk`.`editor_type` AS `modChunk_editor_type`, `modChunk`.`category` AS `modChunk_category`, `modChunk`.`cache_type` AS `modChunk_cache_type`, `modChunk`.`snippet` AS `modChunk_snippet`, `modChunk`.`locked` AS `modChunk_locked`, `modChunk`.`properties` AS `modChunk_properties`, `modChunk`.`static` AS `modChunk_static`, `modChunk`.`static_file` AS `modChunk_static_file`, `Source`.`id` AS `Source_id`, `Source`.`name` AS `Source_name`, `Source`.`description` AS `Source_description`, `Source`.`class_key` AS `Source_class_key`, `Source`.`properties` AS `Source_properties`, `Source`.`is_stream` AS `Source_is_stream` FROM `ewal_site_htmlsnippets` AS `modChunk` LEFT JOIN `ewal_media_sources` `Source` ON `modChunk`.`source` = `Source`.`id` WHERE `modChunk`.`name` = ? ORDER BY `modChunk`.`id` ASC  - 
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:53] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modSession`.`id` AS `modSession_id`, `modSession`.`access` AS `modSession_access`, `modSession`.`data` AS `modSession_data` FROM `ewal_session` AS `modSession` WHERE `modSession`.`id` = ? 
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:53] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 263) Error preparing statement for query: SELECT `modSession`.`id` AS `modSession_id`, `modSession`.`access` AS `modSession_access`, `modSession`.`data` AS `modSession_data` FROM `ewal_session` AS `modSession` WHERE `modSession`.`id` = ?  - 
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:53] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modExtensionPackage`.`id`, `modExtensionPackage`.`namespace`, `modExtensionPackage`.`name`, `modExtensionPackage`.`path`, `modExtensionPackage`.`table_prefix`, `modExtensionPackage`.`service_class`, `modExtensionPackage`.`service_name`, `modExtensionPackage`.`created_at`, `modExtensionPackage`.`updated_at`, Namespace.path AS namespace_path FROM `ewal_extension_packages` AS `modExtensionPackage` JOIN `ewal_namespaces` `Namespace` ON `modExtensionPackage`.`namespace` =  `Namespace`.`name` ORDER BY namespace ASC 
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:53] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modSession`.`id` AS `modSession_id`, `modSession`.`access` AS `modSession_access`, `modSession`.`data` AS `modSession_data` FROM `ewal_session` AS `modSession` WHERE `modSession`.`id` = ? 
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:53] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 263) Error preparing statement for query: SELECT `modSession`.`id` AS `modSession_id`, `modSession`.`access` AS `modSession_access`, `modSession`.`data` AS `modSession_data` FROM `ewal_session` AS `modSession` WHERE `modSession`.`id` = ?  - 
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:53] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `id` FROM `ewal_site_content` AS `modResource` WHERE  ( `modResource`.`context_key` = ? AND `modResource`.`uri` = ? AND `modResource`.`deleted` = ? )  
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:53] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `id` FROM `ewal_site_content` AS `modResource` WHERE  ( `modResource`.`context_key` = ? AND `modResource`.`uri` = ? AND `modResource`.`deleted` = ? )  
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:53] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `id` FROM `ewal_site_content` AS `modResource` WHERE  ( `modResource`.`context_key` = ? AND `modResource`.`uri` = ? AND `modResource`.`deleted` = ? )  
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:53] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modChunk`.`id` AS `modChunk_id`, `modChunk`.`source` AS `modChunk_source`, `modChunk`.`property_preprocess` AS `modChunk_property_preprocess`, `modChunk`.`name` AS `modChunk_name`, `modChunk`.`description` AS `modChunk_description`, `modChunk`.`editor_type` AS `modChunk_editor_type`, `modChunk`.`category` AS `modChunk_category`, `modChunk`.`cache_type` AS `modChunk_cache_type`, `modChunk`.`snippet` AS `modChunk_snippet`, `modChunk`.`locked` AS `modChunk_locked`, `modChunk`.`properties` AS `modChunk_properties`, `modChunk`.`static` AS `modChunk_static`, `modChunk`.`static_file` AS `modChunk_static_file`, `Source`.`id` AS `Source_id`, `Source`.`name` AS `Source_name`, `Source`.`description` AS `Source_description`, `Source`.`class_key` AS `Source_class_key`, `Source`.`properties` AS `Source_properties`, `Source`.`is_stream` AS `Source_is_stream` FROM `ewal_site_htmlsnippets` AS `modChunk` LEFT JOIN `ewal_media_sources` `Source` ON `modChunk`.`source` = `Source`.`id` WHERE `modChunk`.`name` = ? ORDER BY `modChunk`.`id` ASC 
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:53] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 546) Error  preparing statement: SELECT `modChunk`.`id` AS `modChunk_id`, `modChunk`.`source` AS `modChunk_source`, `modChunk`.`property_preprocess` AS `modChunk_property_preprocess`, `modChunk`.`name` AS `modChunk_name`, `modChunk`.`description` AS `modChunk_description`, `modChunk`.`editor_type` AS `modChunk_editor_type`, `modChunk`.`category` AS `modChunk_category`, `modChunk`.`cache_type` AS `modChunk_cache_type`, `modChunk`.`snippet` AS `modChunk_snippet`, `modChunk`.`locked` AS `modChunk_locked`, `modChunk`.`properties` AS `modChunk_properties`, `modChunk`.`static` AS `modChunk_static`, `modChunk`.`static_file` AS `modChunk_static_file`, `Source`.`id` AS `Source_id`, `Source`.`name` AS `Source_name`, `Source`.`description` AS `Source_description`, `Source`.`class_key` AS `Source_class_key`, `Source`.`properties` AS `Source_properties`, `Source`.`is_stream` AS `Source_is_stream` FROM `ewal_site_htmlsnippets` AS `modChunk` LEFT JOIN `ewal_media_sources` `Source` ON `modChunk`.`source` = `Source`.`id` WHERE `modChunk`.`name` = ? ORDER BY `modChunk`.`id` ASC  - 
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:53] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modChunk`.`id` AS `modChunk_id`, `modChunk`.`source` AS `modChunk_source`, `modChunk`.`property_preprocess` AS `modChunk_property_preprocess`, `modChunk`.`name` AS `modChunk_name`, `modChunk`.`description` AS `modChunk_description`, `modChunk`.`editor_type` AS `modChunk_editor_type`, `modChunk`.`category` AS `modChunk_category`, `modChunk`.`cache_type` AS `modChunk_cache_type`, `modChunk`.`snippet` AS `modChunk_snippet`, `modChunk`.`locked` AS `modChunk_locked`, `modChunk`.`properties` AS `modChunk_properties`, `modChunk`.`static` AS `modChunk_static`, `modChunk`.`static_file` AS `modChunk_static_file`, `Source`.`id` AS `Source_id`, `Source`.`name` AS `Source_name`, `Source`.`description` AS `Source_description`, `Source`.`class_key` AS `Source_class_key`, `Source`.`properties` AS `Source_properties`, `Source`.`is_stream` AS `Source_is_stream` FROM `ewal_site_htmlsnippets` AS `modChunk` LEFT JOIN `ewal_media_sources` `Source` ON `modChunk`.`source` = `Source`.`id` WHERE `modChunk`.`name` = ? ORDER BY `modChunk`.`id` ASC 
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:53] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 546) Error  preparing statement: SELECT `modChunk`.`id` AS `modChunk_id`, `modChunk`.`source` AS `modChunk_source`, `modChunk`.`property_preprocess` AS `modChunk_property_preprocess`, `modChunk`.`name` AS `modChunk_name`, `modChunk`.`description` AS `modChunk_description`, `modChunk`.`editor_type` AS `modChunk_editor_type`, `modChunk`.`category` AS `modChunk_category`, `modChunk`.`cache_type` AS `modChunk_cache_type`, `modChunk`.`snippet` AS `modChunk_snippet`, `modChunk`.`locked` AS `modChunk_locked`, `modChunk`.`properties` AS `modChunk_properties`, `modChunk`.`static` AS `modChunk_static`, `modChunk`.`static_file` AS `modChunk_static_file`, `Source`.`id` AS `Source_id`, `Source`.`name` AS `Source_name`, `Source`.`description` AS `Source_description`, `Source`.`class_key` AS `Source_class_key`, `Source`.`properties` AS `Source_properties`, `Source`.`is_stream` AS `Source_is_stream` FROM `ewal_site_htmlsnippets` AS `modChunk` LEFT JOIN `ewal_media_sources` `Source` ON `modChunk`.`source` = `Source`.`id` WHERE `modChunk`.`name` = ? ORDER BY `modChunk`.`id` ASC  - 
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:53] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modChunk`.`id` AS `modChunk_id`, `modChunk`.`source` AS `modChunk_source`, `modChunk`.`property_preprocess` AS `modChunk_property_preprocess`, `modChunk`.`name` AS `modChunk_name`, `modChunk`.`description` AS `modChunk_description`, `modChunk`.`editor_type` AS `modChunk_editor_type`, `modChunk`.`category` AS `modChunk_category`, `modChunk`.`cache_type` AS `modChunk_cache_type`, `modChunk`.`snippet` AS `modChunk_snippet`, `modChunk`.`locked` AS `modChunk_locked`, `modChunk`.`properties` AS `modChunk_properties`, `modChunk`.`static` AS `modChunk_static`, `modChunk`.`static_file` AS `modChunk_static_file`, `Source`.`id` AS `Source_id`, `Source`.`name` AS `Source_name`, `Source`.`description` AS `Source_description`, `Source`.`class_key` AS `Source_class_key`, `Source`.`properties` AS `Source_properties`, `Source`.`is_stream` AS `Source_is_stream` FROM `ewal_site_htmlsnippets` AS `modChunk` LEFT JOIN `ewal_media_sources` `Source` ON `modChunk`.`source` = `Source`.`id` WHERE `modChunk`.`name` = ? ORDER BY `modChunk`.`id` ASC 
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:53] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 546) Error  preparing statement: SELECT `modChunk`.`id` AS `modChunk_id`, `modChunk`.`source` AS `modChunk_source`, `modChunk`.`property_preprocess` AS `modChunk_property_preprocess`, `modChunk`.`name` AS `modChunk_name`, `modChunk`.`description` AS `modChunk_description`, `modChunk`.`editor_type` AS `modChunk_editor_type`, `modChunk`.`category` AS `modChunk_category`, `modChunk`.`cache_type` AS `modChunk_cache_type`, `modChunk`.`snippet` AS `modChunk_snippet`, `modChunk`.`locked` AS `modChunk_locked`, `modChunk`.`properties` AS `modChunk_properties`, `modChunk`.`static` AS `modChunk_static`, `modChunk`.`static_file` AS `modChunk_static_file`, `Source`.`id` AS `Source_id`, `Source`.`name` AS `Source_name`, `Source`.`description` AS `Source_description`, `Source`.`class_key` AS `Source_class_key`, `Source`.`properties` AS `Source_properties`, `Source`.`is_stream` AS `Source_is_stream` FROM `ewal_site_htmlsnippets` AS `modChunk` LEFT JOIN `ewal_media_sources` `Source` ON `modChunk`.`source` = `Source`.`id` WHERE `modChunk`.`name` = ? ORDER BY `modChunk`.`id` ASC  - 
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:53] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modSession`.`id` AS `modSession_id`, `modSession`.`access` AS `modSession_access`, `modSession`.`data` AS `modSession_data` FROM `ewal_session` AS `modSession` WHERE `modSession`.`id` = ? 
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:53] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 263) Error preparing statement for query: SELECT `modSession`.`id` AS `modSession_id`, `modSession`.`access` AS `modSession_access`, `modSession`.`data` AS `modSession_data` FROM `ewal_session` AS `modSession` WHERE `modSession`.`id` = ?  - 
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:53] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modExtensionPackage`.`id`, `modExtensionPackage`.`namespace`, `modExtensionPackage`.`name`, `modExtensionPackage`.`path`, `modExtensionPackage`.`table_prefix`, `modExtensionPackage`.`service_class`, `modExtensionPackage`.`service_name`, `modExtensionPackage`.`created_at`, `modExtensionPackage`.`updated_at`, Namespace.path AS namespace_path FROM `ewal_extension_packages` AS `modExtensionPackage` JOIN `ewal_namespaces` `Namespace` ON `modExtensionPackage`.`namespace` =  `Namespace`.`name` ORDER BY namespace ASC 
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modSession`.`id` AS `modSession_id`, `modSession`.`access` AS `modSession_access`, `modSession`.`data` AS `modSession_data` FROM `ewal_session` AS `modSession` WHERE `modSession`.`id` = ? 
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 263) Error preparing statement for query: SELECT `modSession`.`id` AS `modSession_id`, `modSession`.`access` AS `modSession_access`, `modSession`.`data` AS `modSession_data` FROM `ewal_session` AS `modSession` WHERE `modSession`.`id` = ?  - 
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `id` FROM `ewal_site_content` AS `modResource` WHERE  ( `modResource`.`context_key` = ? AND `modResource`.`uri` = ? AND `modResource`.`deleted` = ? )  
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `id` FROM `ewal_site_content` AS `modResource` WHERE  ( `modResource`.`context_key` = ? AND `modResource`.`uri` = ? AND `modResource`.`deleted` = ? )  
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `id` FROM `ewal_site_content` AS `modResource` WHERE  ( `modResource`.`context_key` = ? AND `modResource`.`uri` = ? AND `modResource`.`deleted` = ? )  
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modChunk`.`id` AS `modChunk_id`, `modChunk`.`source` AS `modChunk_source`, `modChunk`.`property_preprocess` AS `modChunk_property_preprocess`, `modChunk`.`name` AS `modChunk_name`, `modChunk`.`description` AS `modChunk_description`, `modChunk`.`editor_type` AS `modChunk_editor_type`, `modChunk`.`category` AS `modChunk_category`, `modChunk`.`cache_type` AS `modChunk_cache_type`, `modChunk`.`snippet` AS `modChunk_snippet`, `modChunk`.`locked` AS `modChunk_locked`, `modChunk`.`properties` AS `modChunk_properties`, `modChunk`.`static` AS `modChunk_static`, `modChunk`.`static_file` AS `modChunk_static_file`, `Source`.`id` AS `Source_id`, `Source`.`name` AS `Source_name`, `Source`.`description` AS `Source_description`, `Source`.`class_key` AS `Source_class_key`, `Source`.`properties` AS `Source_properties`, `Source`.`is_stream` AS `Source_is_stream` FROM `ewal_site_htmlsnippets` AS `modChunk` LEFT JOIN `ewal_media_sources` `Source` ON `modChunk`.`source` = `Source`.`id` WHERE `modChunk`.`name` = ? ORDER BY `modChunk`.`id` ASC 
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 546) Error  preparing statement: SELECT `modChunk`.`id` AS `modChunk_id`, `modChunk`.`source` AS `modChunk_source`, `modChunk`.`property_preprocess` AS `modChunk_property_preprocess`, `modChunk`.`name` AS `modChunk_name`, `modChunk`.`description` AS `modChunk_description`, `modChunk`.`editor_type` AS `modChunk_editor_type`, `modChunk`.`category` AS `modChunk_category`, `modChunk`.`cache_type` AS `modChunk_cache_type`, `modChunk`.`snippet` AS `modChunk_snippet`, `modChunk`.`locked` AS `modChunk_locked`, `modChunk`.`properties` AS `modChunk_properties`, `modChunk`.`static` AS `modChunk_static`, `modChunk`.`static_file` AS `modChunk_static_file`, `Source`.`id` AS `Source_id`, `Source`.`name` AS `Source_name`, `Source`.`description` AS `Source_description`, `Source`.`class_key` AS `Source_class_key`, `Source`.`properties` AS `Source_properties`, `Source`.`is_stream` AS `Source_is_stream` FROM `ewal_site_htmlsnippets` AS `modChunk` LEFT JOIN `ewal_media_sources` `Source` ON `modChunk`.`source` = `Source`.`id` WHERE `modChunk`.`name` = ? ORDER BY `modChunk`.`id` ASC  - 
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modChunk`.`id` AS `modChunk_id`, `modChunk`.`source` AS `modChunk_source`, `modChunk`.`property_preprocess` AS `modChunk_property_preprocess`, `modChunk`.`name` AS `modChunk_name`, `modChunk`.`description` AS `modChunk_description`, `modChunk`.`editor_type` AS `modChunk_editor_type`, `modChunk`.`category` AS `modChunk_category`, `modChunk`.`cache_type` AS `modChunk_cache_type`, `modChunk`.`snippet` AS `modChunk_snippet`, `modChunk`.`locked` AS `modChunk_locked`, `modChunk`.`properties` AS `modChunk_properties`, `modChunk`.`static` AS `modChunk_static`, `modChunk`.`static_file` AS `modChunk_static_file`, `Source`.`id` AS `Source_id`, `Source`.`name` AS `Source_name`, `Source`.`description` AS `Source_description`, `Source`.`class_key` AS `Source_class_key`, `Source`.`properties` AS `Source_properties`, `Source`.`is_stream` AS `Source_is_stream` FROM `ewal_site_htmlsnippets` AS `modChunk` LEFT JOIN `ewal_media_sources` `Source` ON `modChunk`.`source` = `Source`.`id` WHERE `modChunk`.`name` = ? ORDER BY `modChunk`.`id` ASC 
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 546) Error  preparing statement: SELECT `modChunk`.`id` AS `modChunk_id`, `modChunk`.`source` AS `modChunk_source`, `modChunk`.`property_preprocess` AS `modChunk_property_preprocess`, `modChunk`.`name` AS `modChunk_name`, `modChunk`.`description` AS `modChunk_description`, `modChunk`.`editor_type` AS `modChunk_editor_type`, `modChunk`.`category` AS `modChunk_category`, `modChunk`.`cache_type` AS `modChunk_cache_type`, `modChunk`.`snippet` AS `modChunk_snippet`, `modChunk`.`locked` AS `modChunk_locked`, `modChunk`.`properties` AS `modChunk_properties`, `modChunk`.`static` AS `modChunk_static`, `modChunk`.`static_file` AS `modChunk_static_file`, `Source`.`id` AS `Source_id`, `Source`.`name` AS `Source_name`, `Source`.`description` AS `Source_description`, `Source`.`class_key` AS `Source_class_key`, `Source`.`properties` AS `Source_properties`, `Source`.`is_stream` AS `Source_is_stream` FROM `ewal_site_htmlsnippets` AS `modChunk` LEFT JOIN `ewal_media_sources` `Source` ON `modChunk`.`source` = `Source`.`id` WHERE `modChunk`.`name` = ? ORDER BY `modChunk`.`id` ASC  - 
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modChunk`.`id` AS `modChunk_id`, `modChunk`.`source` AS `modChunk_source`, `modChunk`.`property_preprocess` AS `modChunk_property_preprocess`, `modChunk`.`name` AS `modChunk_name`, `modChunk`.`description` AS `modChunk_description`, `modChunk`.`editor_type` AS `modChunk_editor_type`, `modChunk`.`category` AS `modChunk_category`, `modChunk`.`cache_type` AS `modChunk_cache_type`, `modChunk`.`snippet` AS `modChunk_snippet`, `modChunk`.`locked` AS `modChunk_locked`, `modChunk`.`properties` AS `modChunk_properties`, `modChunk`.`static` AS `modChunk_static`, `modChunk`.`static_file` AS `modChunk_static_file`, `Source`.`id` AS `Source_id`, `Source`.`name` AS `Source_name`, `Source`.`description` AS `Source_description`, `Source`.`class_key` AS `Source_class_key`, `Source`.`properties` AS `Source_properties`, `Source`.`is_stream` AS `Source_is_stream` FROM `ewal_site_htmlsnippets` AS `modChunk` LEFT JOIN `ewal_media_sources` `Source` ON `modChunk`.`source` = `Source`.`id` WHERE `modChunk`.`name` = ? ORDER BY `modChunk`.`id` ASC 
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 546) Error  preparing statement: SELECT `modChunk`.`id` AS `modChunk_id`, `modChunk`.`source` AS `modChunk_source`, `modChunk`.`property_preprocess` AS `modChunk_property_preprocess`, `modChunk`.`name` AS `modChunk_name`, `modChunk`.`description` AS `modChunk_description`, `modChunk`.`editor_type` AS `modChunk_editor_type`, `modChunk`.`category` AS `modChunk_category`, `modChunk`.`cache_type` AS `modChunk_cache_type`, `modChunk`.`snippet` AS `modChunk_snippet`, `modChunk`.`locked` AS `modChunk_locked`, `modChunk`.`properties` AS `modChunk_properties`, `modChunk`.`static` AS `modChunk_static`, `modChunk`.`static_file` AS `modChunk_static_file`, `Source`.`id` AS `Source_id`, `Source`.`name` AS `Source_name`, `Source`.`description` AS `Source_description`, `Source`.`class_key` AS `Source_class_key`, `Source`.`properties` AS `Source_properties`, `Source`.`is_stream` AS `Source_is_stream` FROM `ewal_site_htmlsnippets` AS `modChunk` LEFT JOIN `ewal_media_sources` `Source` ON `modChunk`.`source` = `Source`.`id` WHERE `modChunk`.`name` = ? ORDER BY `modChunk`.`id` ASC  - 
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modSession`.`id` AS `modSession_id`, `modSession`.`access` AS `modSession_access`, `modSession`.`data` AS `modSession_data` FROM `ewal_session` AS `modSession` WHERE `modSession`.`id` = ? 
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 263) Error preparing statement for query: SELECT `modSession`.`id` AS `modSession_id`, `modSession`.`access` AS `modSession_access`, `modSession`.`data` AS `modSession_data` FROM `ewal_session` AS `modSession` WHERE `modSession`.`id` = ?  - 
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modExtensionPackage`.`id`, `modExtensionPackage`.`namespace`, `modExtensionPackage`.`name`, `modExtensionPackage`.`path`, `modExtensionPackage`.`table_prefix`, `modExtensionPackage`.`service_class`, `modExtensionPackage`.`service_name`, `modExtensionPackage`.`created_at`, `modExtensionPackage`.`updated_at`, Namespace.path AS namespace_path FROM `ewal_extension_packages` AS `modExtensionPackage` JOIN `ewal_namespaces` `Namespace` ON `modExtensionPackage`.`namespace` =  `Namespace`.`name` ORDER BY namespace ASC 
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modSession`.`id` AS `modSession_id`, `modSession`.`access` AS `modSession_access`, `modSession`.`data` AS `modSession_data` FROM `ewal_session` AS `modSession` WHERE `modSession`.`id` = ? 
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 263) Error preparing statement for query: SELECT `modSession`.`id` AS `modSession_id`, `modSession`.`access` AS `modSession_access`, `modSession`.`data` AS `modSession_data` FROM `ewal_session` AS `modSession` WHERE `modSession`.`id` = ?  - 
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `id` FROM `ewal_site_content` AS `modResource` WHERE  ( `modResource`.`context_key` = ? AND `modResource`.`uri` = ? AND `modResource`.`deleted` = ? )  
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `id` FROM `ewal_site_content` AS `modResource` WHERE  ( `modResource`.`context_key` = ? AND `modResource`.`uri` = ? AND `modResource`.`deleted` = ? )  
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `id` FROM `ewal_site_content` AS `modResource` WHERE  ( `modResource`.`context_key` = ? AND `modResource`.`uri` = ? AND `modResource`.`deleted` = ? )  
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modChunk`.`id` AS `modChunk_id`, `modChunk`.`source` AS `modChunk_source`, `modChunk`.`property_preprocess` AS `modChunk_property_preprocess`, `modChunk`.`name` AS `modChunk_name`, `modChunk`.`description` AS `modChunk_description`, `modChunk`.`editor_type` AS `modChunk_editor_type`, `modChunk`.`category` AS `modChunk_category`, `modChunk`.`cache_type` AS `modChunk_cache_type`, `modChunk`.`snippet` AS `modChunk_snippet`, `modChunk`.`locked` AS `modChunk_locked`, `modChunk`.`properties` AS `modChunk_properties`, `modChunk`.`static` AS `modChunk_static`, `modChunk`.`static_file` AS `modChunk_static_file`, `Source`.`id` AS `Source_id`, `Source`.`name` AS `Source_name`, `Source`.`description` AS `Source_description`, `Source`.`class_key` AS `Source_class_key`, `Source`.`properties` AS `Source_properties`, `Source`.`is_stream` AS `Source_is_stream` FROM `ewal_site_htmlsnippets` AS `modChunk` LEFT JOIN `ewal_media_sources` `Source` ON `modChunk`.`source` = `Source`.`id` WHERE `modChunk`.`name` = ? ORDER BY `modChunk`.`id` ASC 
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 546) Error  preparing statement: SELECT `modChunk`.`id` AS `modChunk_id`, `modChunk`.`source` AS `modChunk_source`, `modChunk`.`property_preprocess` AS `modChunk_property_preprocess`, `modChunk`.`name` AS `modChunk_name`, `modChunk`.`description` AS `modChunk_description`, `modChunk`.`editor_type` AS `modChunk_editor_type`, `modChunk`.`category` AS `modChunk_category`, `modChunk`.`cache_type` AS `modChunk_cache_type`, `modChunk`.`snippet` AS `modChunk_snippet`, `modChunk`.`locked` AS `modChunk_locked`, `modChunk`.`properties` AS `modChunk_properties`, `modChunk`.`static` AS `modChunk_static`, `modChunk`.`static_file` AS `modChunk_static_file`, `Source`.`id` AS `Source_id`, `Source`.`name` AS `Source_name`, `Source`.`description` AS `Source_description`, `Source`.`class_key` AS `Source_class_key`, `Source`.`properties` AS `Source_properties`, `Source`.`is_stream` AS `Source_is_stream` FROM `ewal_site_htmlsnippets` AS `modChunk` LEFT JOIN `ewal_media_sources` `Source` ON `modChunk`.`source` = `Source`.`id` WHERE `modChunk`.`name` = ? ORDER BY `modChunk`.`id` ASC  - 
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modChunk`.`id` AS `modChunk_id`, `modChunk`.`source` AS `modChunk_source`, `modChunk`.`property_preprocess` AS `modChunk_property_preprocess`, `modChunk`.`name` AS `modChunk_name`, `modChunk`.`description` AS `modChunk_description`, `modChunk`.`editor_type` AS `modChunk_editor_type`, `modChunk`.`category` AS `modChunk_category`, `modChunk`.`cache_type` AS `modChunk_cache_type`, `modChunk`.`snippet` AS `modChunk_snippet`, `modChunk`.`locked` AS `modChunk_locked`, `modChunk`.`properties` AS `modChunk_properties`, `modChunk`.`static` AS `modChunk_static`, `modChunk`.`static_file` AS `modChunk_static_file`, `Source`.`id` AS `Source_id`, `Source`.`name` AS `Source_name`, `Source`.`description` AS `Source_description`, `Source`.`class_key` AS `Source_class_key`, `Source`.`properties` AS `Source_properties`, `Source`.`is_stream` AS `Source_is_stream` FROM `ewal_site_htmlsnippets` AS `modChunk` LEFT JOIN `ewal_media_sources` `Source` ON `modChunk`.`source` = `Source`.`id` WHERE `modChunk`.`name` = ? ORDER BY `modChunk`.`id` ASC 
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 546) Error  preparing statement: SELECT `modChunk`.`id` AS `modChunk_id`, `modChunk`.`source` AS `modChunk_source`, `modChunk`.`property_preprocess` AS `modChunk_property_preprocess`, `modChunk`.`name` AS `modChunk_name`, `modChunk`.`description` AS `modChunk_description`, `modChunk`.`editor_type` AS `modChunk_editor_type`, `modChunk`.`category` AS `modChunk_category`, `modChunk`.`cache_type` AS `modChunk_cache_type`, `modChunk`.`snippet` AS `modChunk_snippet`, `modChunk`.`locked` AS `modChunk_locked`, `modChunk`.`properties` AS `modChunk_properties`, `modChunk`.`static` AS `modChunk_static`, `modChunk`.`static_file` AS `modChunk_static_file`, `Source`.`id` AS `Source_id`, `Source`.`name` AS `Source_name`, `Source`.`description` AS `Source_description`, `Source`.`class_key` AS `Source_class_key`, `Source`.`properties` AS `Source_properties`, `Source`.`is_stream` AS `Source_is_stream` FROM `ewal_site_htmlsnippets` AS `modChunk` LEFT JOIN `ewal_media_sources` `Source` ON `modChunk`.`source` = `Source`.`id` WHERE `modChunk`.`name` = ? ORDER BY `modChunk`.`id` ASC  - 
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modChunk`.`id` AS `modChunk_id`, `modChunk`.`source` AS `modChunk_source`, `modChunk`.`property_preprocess` AS `modChunk_property_preprocess`, `modChunk`.`name` AS `modChunk_name`, `modChunk`.`description` AS `modChunk_description`, `modChunk`.`editor_type` AS `modChunk_editor_type`, `modChunk`.`category` AS `modChunk_category`, `modChunk`.`cache_type` AS `modChunk_cache_type`, `modChunk`.`snippet` AS `modChunk_snippet`, `modChunk`.`locked` AS `modChunk_locked`, `modChunk`.`properties` AS `modChunk_properties`, `modChunk`.`static` AS `modChunk_static`, `modChunk`.`static_file` AS `modChunk_static_file`, `Source`.`id` AS `Source_id`, `Source`.`name` AS `Source_name`, `Source`.`description` AS `Source_description`, `Source`.`class_key` AS `Source_class_key`, `Source`.`properties` AS `Source_properties`, `Source`.`is_stream` AS `Source_is_stream` FROM `ewal_site_htmlsnippets` AS `modChunk` LEFT JOIN `ewal_media_sources` `Source` ON `modChunk`.`source` = `Source`.`id` WHERE `modChunk`.`name` = ? ORDER BY `modChunk`.`id` ASC 
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 546) Error  preparing statement: SELECT `modChunk`.`id` AS `modChunk_id`, `modChunk`.`source` AS `modChunk_source`, `modChunk`.`property_preprocess` AS `modChunk_property_preprocess`, `modChunk`.`name` AS `modChunk_name`, `modChunk`.`description` AS `modChunk_description`, `modChunk`.`editor_type` AS `modChunk_editor_type`, `modChunk`.`category` AS `modChunk_category`, `modChunk`.`cache_type` AS `modChunk_cache_type`, `modChunk`.`snippet` AS `modChunk_snippet`, `modChunk`.`locked` AS `modChunk_locked`, `modChunk`.`properties` AS `modChunk_properties`, `modChunk`.`static` AS `modChunk_static`, `modChunk`.`static_file` AS `modChunk_static_file`, `Source`.`id` AS `Source_id`, `Source`.`name` AS `Source_name`, `Source`.`description` AS `Source_description`, `Source`.`class_key` AS `Source_class_key`, `Source`.`properties` AS `Source_properties`, `Source`.`is_stream` AS `Source_is_stream` FROM `ewal_site_htmlsnippets` AS `modChunk` LEFT JOIN `ewal_media_sources` `Source` ON `modChunk`.`source` = `Source`.`id` WHERE `modChunk`.`name` = ? ORDER BY `modChunk`.`id` ASC  - 
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modSession`.`id` AS `modSession_id`, `modSession`.`access` AS `modSession_access`, `modSession`.`data` AS `modSession_data` FROM `ewal_session` AS `modSession` WHERE `modSession`.`id` = ? 
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 263) Error preparing statement for query: SELECT `modSession`.`id` AS `modSession_id`, `modSession`.`access` AS `modSession_access`, `modSession`.`data` AS `modSession_data` FROM `ewal_session` AS `modSession` WHERE `modSession`.`id` = ?  - 
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:32:54] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modExtensionPackage`.`id`, `modExtensionPackage`.`namespace`, `modExtensionPackage`.`name`, `modExtensionPackage`.`path`, `modExtensionPackage`.`table_prefix`, `modExtensionPackage`.`service_class`, `modExtensionPackage`.`service_name`, `modExtensionPackage`.`created_at`, `modExtensionPackage`.`updated_at`, Namespace.path AS namespace_path FROM `ewal_extension_packages` AS `modExtensionPackage` JOIN `ewal_namespaces` `Namespace` ON `modExtensionPackage`.`namespace` =  `Namespace`.`name` ORDER BY namespace ASC 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modSession`.`id` AS `modSession_id`, `modSession`.`access` AS `modSession_access`, `modSession`.`data` AS `modSession_data` FROM `ewal_session` AS `modSession` WHERE `modSession`.`id` = ? 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 263) Error preparing statement for query: SELECT `modSession`.`id` AS `modSession_id`, `modSession`.`access` AS `modSession_access`, `modSession`.`data` AS `modSession_data` FROM `ewal_session` AS `modSession` WHERE `modSession`.`id` = ?  - 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modLexiconEntry`.`id` AS `modLexiconEntry_id`, `modLexiconEntry`.`name` AS `modLexiconEntry_name`, `modLexiconEntry`.`value` AS `modLexiconEntry_value`, `modLexiconEntry`.`topic` AS `modLexiconEntry_topic`, `modLexiconEntry`.`namespace` AS `modLexiconEntry_namespace`, `modLexiconEntry`.`language` AS `modLexiconEntry_language`, `modLexiconEntry`.`createdon` AS `modLexiconEntry_createdon`, `modLexiconEntry`.`editedon` AS `modLexiconEntry_editedon` FROM `ewal_lexicon_entries` AS `modLexiconEntry` JOIN `ewal_namespaces` `Namespace` ON `modLexiconEntry`.`namespace` =  `Namespace`.`name` WHERE  ( `modLexiconEntry`.`topic` = ? AND `modLexiconEntry`.`language` = ? AND `Namespace`.`name` = ? )  ORDER BY `modLexiconEntry`.`name` ASC 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 263) Error preparing statement for query: SELECT `modLexiconEntry`.`id` AS `modLexiconEntry_id`, `modLexiconEntry`.`name` AS `modLexiconEntry_name`, `modLexiconEntry`.`value` AS `modLexiconEntry_value`, `modLexiconEntry`.`topic` AS `modLexiconEntry_topic`, `modLexiconEntry`.`namespace` AS `modLexiconEntry_namespace`, `modLexiconEntry`.`language` AS `modLexiconEntry_language`, `modLexiconEntry`.`createdon` AS `modLexiconEntry_createdon`, `modLexiconEntry`.`editedon` AS `modLexiconEntry_editedon` FROM `ewal_lexicon_entries` AS `modLexiconEntry` JOIN `ewal_namespaces` `Namespace` ON `modLexiconEntry`.`namespace` =  `Namespace`.`name` WHERE  ( `modLexiconEntry`.`topic` = ? AND `modLexiconEntry`.`language` = ? AND `Namespace`.`name` = ? )  ORDER BY `modLexiconEntry`.`name` ASC  - 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modLexiconEntry`.`id` AS `modLexiconEntry_id`, `modLexiconEntry`.`name` AS `modLexiconEntry_name`, `modLexiconEntry`.`value` AS `modLexiconEntry_value`, `modLexiconEntry`.`topic` AS `modLexiconEntry_topic`, `modLexiconEntry`.`namespace` AS `modLexiconEntry_namespace`, `modLexiconEntry`.`language` AS `modLexiconEntry_language`, `modLexiconEntry`.`createdon` AS `modLexiconEntry_createdon`, `modLexiconEntry`.`editedon` AS `modLexiconEntry_editedon` FROM `ewal_lexicon_entries` AS `modLexiconEntry` JOIN `ewal_namespaces` `Namespace` ON `modLexiconEntry`.`namespace` =  `Namespace`.`name` WHERE  ( `modLexiconEntry`.`topic` = ? AND `modLexiconEntry`.`language` = ? AND `Namespace`.`name` = ? )  ORDER BY `modLexiconEntry`.`name` ASC 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 263) Error preparing statement for query: SELECT `modLexiconEntry`.`id` AS `modLexiconEntry_id`, `modLexiconEntry`.`name` AS `modLexiconEntry_name`, `modLexiconEntry`.`value` AS `modLexiconEntry_value`, `modLexiconEntry`.`topic` AS `modLexiconEntry_topic`, `modLexiconEntry`.`namespace` AS `modLexiconEntry_namespace`, `modLexiconEntry`.`language` AS `modLexiconEntry_language`, `modLexiconEntry`.`createdon` AS `modLexiconEntry_createdon`, `modLexiconEntry`.`editedon` AS `modLexiconEntry_editedon` FROM `ewal_lexicon_entries` AS `modLexiconEntry` JOIN `ewal_namespaces` `Namespace` ON `modLexiconEntry`.`namespace` =  `Namespace`.`name` WHERE  ( `modLexiconEntry`.`topic` = ? AND `modLexiconEntry`.`language` = ? AND `Namespace`.`name` = ? )  ORDER BY `modLexiconEntry`.`name` ASC  - 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modLexiconEntry`.`id` AS `modLexiconEntry_id`, `modLexiconEntry`.`name` AS `modLexiconEntry_name`, `modLexiconEntry`.`value` AS `modLexiconEntry_value`, `modLexiconEntry`.`topic` AS `modLexiconEntry_topic`, `modLexiconEntry`.`namespace` AS `modLexiconEntry_namespace`, `modLexiconEntry`.`language` AS `modLexiconEntry_language`, `modLexiconEntry`.`createdon` AS `modLexiconEntry_createdon`, `modLexiconEntry`.`editedon` AS `modLexiconEntry_editedon` FROM `ewal_lexicon_entries` AS `modLexiconEntry` JOIN `ewal_namespaces` `Namespace` ON `modLexiconEntry`.`namespace` =  `Namespace`.`name` WHERE  ( `modLexiconEntry`.`topic` = ? AND `modLexiconEntry`.`language` = ? AND `Namespace`.`name` = ? )  ORDER BY `modLexiconEntry`.`name` ASC 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 263) Error preparing statement for query: SELECT `modLexiconEntry`.`id` AS `modLexiconEntry_id`, `modLexiconEntry`.`name` AS `modLexiconEntry_name`, `modLexiconEntry`.`value` AS `modLexiconEntry_value`, `modLexiconEntry`.`topic` AS `modLexiconEntry_topic`, `modLexiconEntry`.`namespace` AS `modLexiconEntry_namespace`, `modLexiconEntry`.`language` AS `modLexiconEntry_language`, `modLexiconEntry`.`createdon` AS `modLexiconEntry_createdon`, `modLexiconEntry`.`editedon` AS `modLexiconEntry_editedon` FROM `ewal_lexicon_entries` AS `modLexiconEntry` JOIN `ewal_namespaces` `Namespace` ON `modLexiconEntry`.`namespace` =  `Namespace`.`name` WHERE  ( `modLexiconEntry`.`topic` = ? AND `modLexiconEntry`.`language` = ? AND `Namespace`.`name` = ? )  ORDER BY `modLexiconEntry`.`name` ASC  - 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modLexiconEntry`.`id` AS `modLexiconEntry_id`, `modLexiconEntry`.`name` AS `modLexiconEntry_name`, `modLexiconEntry`.`value` AS `modLexiconEntry_value`, `modLexiconEntry`.`topic` AS `modLexiconEntry_topic`, `modLexiconEntry`.`namespace` AS `modLexiconEntry_namespace`, `modLexiconEntry`.`language` AS `modLexiconEntry_language`, `modLexiconEntry`.`createdon` AS `modLexiconEntry_createdon`, `modLexiconEntry`.`editedon` AS `modLexiconEntry_editedon` FROM `ewal_lexicon_entries` AS `modLexiconEntry` JOIN `ewal_namespaces` `Namespace` ON `modLexiconEntry`.`namespace` =  `Namespace`.`name` WHERE  ( `modLexiconEntry`.`topic` = ? AND `modLexiconEntry`.`language` = ? AND `Namespace`.`name` = ? )  ORDER BY `modLexiconEntry`.`name` ASC 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 263) Error preparing statement for query: SELECT `modLexiconEntry`.`id` AS `modLexiconEntry_id`, `modLexiconEntry`.`name` AS `modLexiconEntry_name`, `modLexiconEntry`.`value` AS `modLexiconEntry_value`, `modLexiconEntry`.`topic` AS `modLexiconEntry_topic`, `modLexiconEntry`.`namespace` AS `modLexiconEntry_namespace`, `modLexiconEntry`.`language` AS `modLexiconEntry_language`, `modLexiconEntry`.`createdon` AS `modLexiconEntry_createdon`, `modLexiconEntry`.`editedon` AS `modLexiconEntry_editedon` FROM `ewal_lexicon_entries` AS `modLexiconEntry` JOIN `ewal_namespaces` `Namespace` ON `modLexiconEntry`.`namespace` =  `Namespace`.`name` WHERE  ( `modLexiconEntry`.`topic` = ? AND `modLexiconEntry`.`language` = ? AND `Namespace`.`name` = ? )  ORDER BY `modLexiconEntry`.`name` ASC  - 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT DISTINCT `language` FROM `ewal_lexicon_entries` AS `modLexiconEntry` WHERE  ( `modLexiconEntry`.`namespace` = ? AND `modLexiconEntry`.`language` NOT IN (,,,,,,,,,,,,,,,,,,,,,,,,,,,,,) )  
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modLexiconEntry`.`id` AS `modLexiconEntry_id`, `modLexiconEntry`.`name` AS `modLexiconEntry_name`, `modLexiconEntry`.`value` AS `modLexiconEntry_value`, `modLexiconEntry`.`topic` AS `modLexiconEntry_topic`, `modLexiconEntry`.`namespace` AS `modLexiconEntry_namespace`, `modLexiconEntry`.`language` AS `modLexiconEntry_language`, `modLexiconEntry`.`createdon` AS `modLexiconEntry_createdon`, `modLexiconEntry`.`editedon` AS `modLexiconEntry_editedon` FROM `ewal_lexicon_entries` AS `modLexiconEntry` JOIN `ewal_namespaces` `Namespace` ON `modLexiconEntry`.`namespace` =  `Namespace`.`name` WHERE  ( `modLexiconEntry`.`topic` = ? AND `modLexiconEntry`.`language` = ? AND `Namespace`.`name` = ? )  ORDER BY `modLexiconEntry`.`name` ASC 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 263) Error preparing statement for query: SELECT `modLexiconEntry`.`id` AS `modLexiconEntry_id`, `modLexiconEntry`.`name` AS `modLexiconEntry_name`, `modLexiconEntry`.`value` AS `modLexiconEntry_value`, `modLexiconEntry`.`topic` AS `modLexiconEntry_topic`, `modLexiconEntry`.`namespace` AS `modLexiconEntry_namespace`, `modLexiconEntry`.`language` AS `modLexiconEntry_language`, `modLexiconEntry`.`createdon` AS `modLexiconEntry_createdon`, `modLexiconEntry`.`editedon` AS `modLexiconEntry_editedon` FROM `ewal_lexicon_entries` AS `modLexiconEntry` JOIN `ewal_namespaces` `Namespace` ON `modLexiconEntry`.`namespace` =  `Namespace`.`name` WHERE  ( `modLexiconEntry`.`topic` = ? AND `modLexiconEntry`.`language` = ? AND `Namespace`.`name` = ? )  ORDER BY `modLexiconEntry`.`name` ASC  - 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modActionDom`.`id`, `modActionDom`.`set`, `modActionDom`.`action`, `modActionDom`.`name`, `modActionDom`.`description`, `modActionDom`.`xtype`, `modActionDom`.`container`, `modActionDom`.`rule`, `modActionDom`.`value`, `modActionDom`.`constraint`, `modActionDom`.`constraint_field`, `modActionDom`.`constraint_class`, `modActionDom`.`active`, `modActionDom`.`for_parent`, `modActionDom`.`rank`, `FCSet`.`constraint_class`, `FCSet`.`constraint_field`, `FCSet`.`constraint`, `FCSet`.`template` FROM `ewal_actiondom` AS `modActionDom` JOIN `ewal_fc_sets` `FCSet` ON `modActionDom`.`set` =  `FCSet`.`id` JOIN `ewal_fc_profiles` `Profile` ON FCSet.profile = Profile.id LEFT JOIN `ewal_fc_profiles_usergroups` `ProfileUserGroup` ON Profile.id = ProfileUserGroup.profile LEFT JOIN `ewal_fc_profiles` `UGProfile` ON UGProfile.id = ProfileUserGroup.profile WHERE  ( `modActionDom`.`action` = ? AND `modActionDom`.`for_parent` = ? AND `FCSet`.`active` = ? AND `Profile`.`active` = ? )  ORDER BY modActionDom.rank ASC 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 263) Error preparing statement for query: SELECT `modActionDom`.`id`, `modActionDom`.`set`, `modActionDom`.`action`, `modActionDom`.`name`, `modActionDom`.`description`, `modActionDom`.`xtype`, `modActionDom`.`container`, `modActionDom`.`rule`, `modActionDom`.`value`, `modActionDom`.`constraint`, `modActionDom`.`constraint_field`, `modActionDom`.`constraint_class`, `modActionDom`.`active`, `modActionDom`.`for_parent`, `modActionDom`.`rank`, `FCSet`.`constraint_class`, `FCSet`.`constraint_field`, `FCSet`.`constraint`, `FCSet`.`template` FROM `ewal_actiondom` AS `modActionDom` JOIN `ewal_fc_sets` `FCSet` ON `modActionDom`.`set` =  `FCSet`.`id` JOIN `ewal_fc_profiles` `Profile` ON FCSet.profile = Profile.id LEFT JOIN `ewal_fc_profiles_usergroups` `ProfileUserGroup` ON Profile.id = ProfileUserGroup.profile LEFT JOIN `ewal_fc_profiles` `UGProfile` ON UGProfile.id = ProfileUserGroup.profile WHERE  ( `modActionDom`.`action` = ? AND `modActionDom`.`for_parent` = ? AND `FCSet`.`active` = ? AND `Profile`.`active` = ? )  ORDER BY modActionDom.rank ASC  - 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modNamespace`.`name` AS `modNamespace_name`, `modNamespace`.`path` AS `modNamespace_path`, `modNamespace`.`assets_path` AS `modNamespace_assets_path` FROM `ewal_namespaces` AS `modNamespace` WHERE `modNamespace`.`name` = ? 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 263) Error preparing statement for query: SELECT `modNamespace`.`name` AS `modNamespace_name`, `modNamespace`.`path` AS `modNamespace_path`, `modNamespace`.`assets_path` AS `modNamespace_assets_path` FROM `ewal_namespaces` AS `modNamespace` WHERE `modNamespace`.`name` = ?  - 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modLexiconEntry`.`id` AS `modLexiconEntry_id`, `modLexiconEntry`.`name` AS `modLexiconEntry_name`, `modLexiconEntry`.`value` AS `modLexiconEntry_value`, `modLexiconEntry`.`topic` AS `modLexiconEntry_topic`, `modLexiconEntry`.`namespace` AS `modLexiconEntry_namespace`, `modLexiconEntry`.`language` AS `modLexiconEntry_language`, `modLexiconEntry`.`createdon` AS `modLexiconEntry_createdon`, `modLexiconEntry`.`editedon` AS `modLexiconEntry_editedon` FROM `ewal_lexicon_entries` AS `modLexiconEntry` JOIN `ewal_namespaces` `Namespace` ON `modLexiconEntry`.`namespace` =  `Namespace`.`name` WHERE  ( `modLexiconEntry`.`topic` = ? AND `modLexiconEntry`.`language` = ? AND `Namespace`.`name` = ? )  ORDER BY `modLexiconEntry`.`name` ASC 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 263) Error preparing statement for query: SELECT `modLexiconEntry`.`id` AS `modLexiconEntry_id`, `modLexiconEntry`.`name` AS `modLexiconEntry_name`, `modLexiconEntry`.`value` AS `modLexiconEntry_value`, `modLexiconEntry`.`topic` AS `modLexiconEntry_topic`, `modLexiconEntry`.`namespace` AS `modLexiconEntry_namespace`, `modLexiconEntry`.`language` AS `modLexiconEntry_language`, `modLexiconEntry`.`createdon` AS `modLexiconEntry_createdon`, `modLexiconEntry`.`editedon` AS `modLexiconEntry_editedon` FROM `ewal_lexicon_entries` AS `modLexiconEntry` JOIN `ewal_namespaces` `Namespace` ON `modLexiconEntry`.`namespace` =  `Namespace`.`name` WHERE  ( `modLexiconEntry`.`topic` = ? AND `modLexiconEntry`.`language` = ? AND `Namespace`.`name` = ? )  ORDER BY `modLexiconEntry`.`name` ASC  - 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modNamespace`.`name` AS `modNamespace_name`, `modNamespace`.`path` AS `modNamespace_path`, `modNamespace`.`assets_path` AS `modNamespace_assets_path` FROM `ewal_namespaces` AS `modNamespace` WHERE `modNamespace`.`name` = ? 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 263) Error preparing statement for query: SELECT `modNamespace`.`name` AS `modNamespace_name`, `modNamespace`.`path` AS `modNamespace_path`, `modNamespace`.`assets_path` AS `modNamespace_assets_path` FROM `ewal_namespaces` AS `modNamespace` WHERE `modNamespace`.`name` = ?  - 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modLexiconEntry`.`id` AS `modLexiconEntry_id`, `modLexiconEntry`.`name` AS `modLexiconEntry_name`, `modLexiconEntry`.`value` AS `modLexiconEntry_value`, `modLexiconEntry`.`topic` AS `modLexiconEntry_topic`, `modLexiconEntry`.`namespace` AS `modLexiconEntry_namespace`, `modLexiconEntry`.`language` AS `modLexiconEntry_language`, `modLexiconEntry`.`createdon` AS `modLexiconEntry_createdon`, `modLexiconEntry`.`editedon` AS `modLexiconEntry_editedon` FROM `ewal_lexicon_entries` AS `modLexiconEntry` JOIN `ewal_namespaces` `Namespace` ON `modLexiconEntry`.`namespace` =  `Namespace`.`name` WHERE  ( `modLexiconEntry`.`topic` = ? AND `modLexiconEntry`.`language` = ? AND `Namespace`.`name` = ? )  ORDER BY `modLexiconEntry`.`name` ASC 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 263) Error preparing statement for query: SELECT `modLexiconEntry`.`id` AS `modLexiconEntry_id`, `modLexiconEntry`.`name` AS `modLexiconEntry_name`, `modLexiconEntry`.`value` AS `modLexiconEntry_value`, `modLexiconEntry`.`topic` AS `modLexiconEntry_topic`, `modLexiconEntry`.`namespace` AS `modLexiconEntry_namespace`, `modLexiconEntry`.`language` AS `modLexiconEntry_language`, `modLexiconEntry`.`createdon` AS `modLexiconEntry_createdon`, `modLexiconEntry`.`editedon` AS `modLexiconEntry_editedon` FROM `ewal_lexicon_entries` AS `modLexiconEntry` JOIN `ewal_namespaces` `Namespace` ON `modLexiconEntry`.`namespace` =  `Namespace`.`name` WHERE  ( `modLexiconEntry`.`topic` = ? AND `modLexiconEntry`.`language` = ? AND `Namespace`.`name` = ? )  ORDER BY `modLexiconEntry`.`name` ASC  - 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modSession`.`id` AS `modSession_id`, `modSession`.`access` AS `modSession_access`, `modSession`.`data` AS `modSession_data` FROM `ewal_session` AS `modSession` WHERE `modSession`.`id` = ? 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 263) Error preparing statement for query: SELECT `modSession`.`id` AS `modSession_id`, `modSession`.`access` AS `modSession_access`, `modSession`.`data` AS `modSession_data` FROM `ewal_session` AS `modSession` WHERE `modSession`.`id` = ?  - 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modExtensionPackage`.`id`, `modExtensionPackage`.`namespace`, `modExtensionPackage`.`name`, `modExtensionPackage`.`path`, `modExtensionPackage`.`table_prefix`, `modExtensionPackage`.`service_class`, `modExtensionPackage`.`service_name`, `modExtensionPackage`.`created_at`, `modExtensionPackage`.`updated_at`, Namespace.path AS namespace_path FROM `ewal_extension_packages` AS `modExtensionPackage` JOIN `ewal_namespaces` `Namespace` ON `modExtensionPackage`.`namespace` =  `Namespace`.`name` ORDER BY namespace ASC 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modSession`.`id` AS `modSession_id`, `modSession`.`access` AS `modSession_access`, `modSession`.`data` AS `modSession_data` FROM `ewal_session` AS `modSession` WHERE `modSession`.`id` = ? 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 263) Error preparing statement for query: SELECT `modSession`.`id` AS `modSession_id`, `modSession`.`access` AS `modSession_access`, `modSession`.`data` AS `modSession_data` FROM `ewal_session` AS `modSession` WHERE `modSession`.`id` = ?  - 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `id` FROM `ewal_site_content` AS `modResource` WHERE  ( `modResource`.`context_key` = ? AND `modResource`.`uri` = ? AND `modResource`.`deleted` = ? )  
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `id` FROM `ewal_site_content` AS `modResource` WHERE  ( `modResource`.`context_key` = ? AND `modResource`.`uri` = ? AND `modResource`.`deleted` = ? )  
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `id` FROM `ewal_site_content` AS `modResource` WHERE  ( `modResource`.`context_key` = ? AND `modResource`.`uri` = ? AND `modResource`.`deleted` = ? )  
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modChunk`.`id` AS `modChunk_id`, `modChunk`.`source` AS `modChunk_source`, `modChunk`.`property_preprocess` AS `modChunk_property_preprocess`, `modChunk`.`name` AS `modChunk_name`, `modChunk`.`description` AS `modChunk_description`, `modChunk`.`editor_type` AS `modChunk_editor_type`, `modChunk`.`category` AS `modChunk_category`, `modChunk`.`cache_type` AS `modChunk_cache_type`, `modChunk`.`snippet` AS `modChunk_snippet`, `modChunk`.`locked` AS `modChunk_locked`, `modChunk`.`properties` AS `modChunk_properties`, `modChunk`.`static` AS `modChunk_static`, `modChunk`.`static_file` AS `modChunk_static_file`, `Source`.`id` AS `Source_id`, `Source`.`name` AS `Source_name`, `Source`.`description` AS `Source_description`, `Source`.`class_key` AS `Source_class_key`, `Source`.`properties` AS `Source_properties`, `Source`.`is_stream` AS `Source_is_stream` FROM `ewal_site_htmlsnippets` AS `modChunk` LEFT JOIN `ewal_media_sources` `Source` ON `modChunk`.`source` = `Source`.`id` WHERE `modChunk`.`name` = ? ORDER BY `modChunk`.`id` ASC 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 546) Error  preparing statement: SELECT `modChunk`.`id` AS `modChunk_id`, `modChunk`.`source` AS `modChunk_source`, `modChunk`.`property_preprocess` AS `modChunk_property_preprocess`, `modChunk`.`name` AS `modChunk_name`, `modChunk`.`description` AS `modChunk_description`, `modChunk`.`editor_type` AS `modChunk_editor_type`, `modChunk`.`category` AS `modChunk_category`, `modChunk`.`cache_type` AS `modChunk_cache_type`, `modChunk`.`snippet` AS `modChunk_snippet`, `modChunk`.`locked` AS `modChunk_locked`, `modChunk`.`properties` AS `modChunk_properties`, `modChunk`.`static` AS `modChunk_static`, `modChunk`.`static_file` AS `modChunk_static_file`, `Source`.`id` AS `Source_id`, `Source`.`name` AS `Source_name`, `Source`.`description` AS `Source_description`, `Source`.`class_key` AS `Source_class_key`, `Source`.`properties` AS `Source_properties`, `Source`.`is_stream` AS `Source_is_stream` FROM `ewal_site_htmlsnippets` AS `modChunk` LEFT JOIN `ewal_media_sources` `Source` ON `modChunk`.`source` = `Source`.`id` WHERE `modChunk`.`name` = ? ORDER BY `modChunk`.`id` ASC  - 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modChunk`.`id` AS `modChunk_id`, `modChunk`.`source` AS `modChunk_source`, `modChunk`.`property_preprocess` AS `modChunk_property_preprocess`, `modChunk`.`name` AS `modChunk_name`, `modChunk`.`description` AS `modChunk_description`, `modChunk`.`editor_type` AS `modChunk_editor_type`, `modChunk`.`category` AS `modChunk_category`, `modChunk`.`cache_type` AS `modChunk_cache_type`, `modChunk`.`snippet` AS `modChunk_snippet`, `modChunk`.`locked` AS `modChunk_locked`, `modChunk`.`properties` AS `modChunk_properties`, `modChunk`.`static` AS `modChunk_static`, `modChunk`.`static_file` AS `modChunk_static_file`, `Source`.`id` AS `Source_id`, `Source`.`name` AS `Source_name`, `Source`.`description` AS `Source_description`, `Source`.`class_key` AS `Source_class_key`, `Source`.`properties` AS `Source_properties`, `Source`.`is_stream` AS `Source_is_stream` FROM `ewal_site_htmlsnippets` AS `modChunk` LEFT JOIN `ewal_media_sources` `Source` ON `modChunk`.`source` = `Source`.`id` WHERE `modChunk`.`name` = ? ORDER BY `modChunk`.`id` ASC 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 546) Error  preparing statement: SELECT `modChunk`.`id` AS `modChunk_id`, `modChunk`.`source` AS `modChunk_source`, `modChunk`.`property_preprocess` AS `modChunk_property_preprocess`, `modChunk`.`name` AS `modChunk_name`, `modChunk`.`description` AS `modChunk_description`, `modChunk`.`editor_type` AS `modChunk_editor_type`, `modChunk`.`category` AS `modChunk_category`, `modChunk`.`cache_type` AS `modChunk_cache_type`, `modChunk`.`snippet` AS `modChunk_snippet`, `modChunk`.`locked` AS `modChunk_locked`, `modChunk`.`properties` AS `modChunk_properties`, `modChunk`.`static` AS `modChunk_static`, `modChunk`.`static_file` AS `modChunk_static_file`, `Source`.`id` AS `Source_id`, `Source`.`name` AS `Source_name`, `Source`.`description` AS `Source_description`, `Source`.`class_key` AS `Source_class_key`, `Source`.`properties` AS `Source_properties`, `Source`.`is_stream` AS `Source_is_stream` FROM `ewal_site_htmlsnippets` AS `modChunk` LEFT JOIN `ewal_media_sources` `Source` ON `modChunk`.`source` = `Source`.`id` WHERE `modChunk`.`name` = ? ORDER BY `modChunk`.`id` ASC  - 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modChunk`.`id` AS `modChunk_id`, `modChunk`.`source` AS `modChunk_source`, `modChunk`.`property_preprocess` AS `modChunk_property_preprocess`, `modChunk`.`name` AS `modChunk_name`, `modChunk`.`description` AS `modChunk_description`, `modChunk`.`editor_type` AS `modChunk_editor_type`, `modChunk`.`category` AS `modChunk_category`, `modChunk`.`cache_type` AS `modChunk_cache_type`, `modChunk`.`snippet` AS `modChunk_snippet`, `modChunk`.`locked` AS `modChunk_locked`, `modChunk`.`properties` AS `modChunk_properties`, `modChunk`.`static` AS `modChunk_static`, `modChunk`.`static_file` AS `modChunk_static_file`, `Source`.`id` AS `Source_id`, `Source`.`name` AS `Source_name`, `Source`.`description` AS `Source_description`, `Source`.`class_key` AS `Source_class_key`, `Source`.`properties` AS `Source_properties`, `Source`.`is_stream` AS `Source_is_stream` FROM `ewal_site_htmlsnippets` AS `modChunk` LEFT JOIN `ewal_media_sources` `Source` ON `modChunk`.`source` = `Source`.`id` WHERE `modChunk`.`name` = ? ORDER BY `modChunk`.`id` ASC 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 546) Error  preparing statement: SELECT `modChunk`.`id` AS `modChunk_id`, `modChunk`.`source` AS `modChunk_source`, `modChunk`.`property_preprocess` AS `modChunk_property_preprocess`, `modChunk`.`name` AS `modChunk_name`, `modChunk`.`description` AS `modChunk_description`, `modChunk`.`editor_type` AS `modChunk_editor_type`, `modChunk`.`category` AS `modChunk_category`, `modChunk`.`cache_type` AS `modChunk_cache_type`, `modChunk`.`snippet` AS `modChunk_snippet`, `modChunk`.`locked` AS `modChunk_locked`, `modChunk`.`properties` AS `modChunk_properties`, `modChunk`.`static` AS `modChunk_static`, `modChunk`.`static_file` AS `modChunk_static_file`, `Source`.`id` AS `Source_id`, `Source`.`name` AS `Source_name`, `Source`.`description` AS `Source_description`, `Source`.`class_key` AS `Source_class_key`, `Source`.`properties` AS `Source_properties`, `Source`.`is_stream` AS `Source_is_stream` FROM `ewal_site_htmlsnippets` AS `modChunk` LEFT JOIN `ewal_media_sources` `Source` ON `modChunk`.`source` = `Source`.`id` WHERE `modChunk`.`name` = ? ORDER BY `modChunk`.`id` ASC  - 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modSession`.`id` AS `modSession_id`, `modSession`.`access` AS `modSession_access`, `modSession`.`data` AS `modSession_data` FROM `ewal_session` AS `modSession` WHERE `modSession`.`id` = ? 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 263) Error preparing statement for query: SELECT `modSession`.`id` AS `modSession_id`, `modSession`.`access` AS `modSession_access`, `modSession`.`data` AS `modSession_data` FROM `ewal_session` AS `modSession` WHERE `modSession`.`id` = ?  - 
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:37] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modExtensionPackage`.`id`, `modExtensionPackage`.`namespace`, `modExtensionPackage`.`name`, `modExtensionPackage`.`path`, `modExtensionPackage`.`table_prefix`, `modExtensionPackage`.`service_class`, `modExtensionPackage`.`service_name`, `modExtensionPackage`.`created_at`, `modExtensionPackage`.`updated_at`, Namespace.path AS namespace_path FROM `ewal_extension_packages` AS `modExtensionPackage` JOIN `ewal_namespaces` `Namespace` ON `modExtensionPackage`.`namespace` =  `Namespace`.`name` ORDER BY namespace ASC 
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modSession`.`id` AS `modSession_id`, `modSession`.`access` AS `modSession_access`, `modSession`.`data` AS `modSession_data` FROM `ewal_session` AS `modSession` WHERE `modSession`.`id` = ? 
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 263) Error preparing statement for query: SELECT `modSession`.`id` AS `modSession_id`, `modSession`.`access` AS `modSession_access`, `modSession`.`data` AS `modSession_data` FROM `ewal_session` AS `modSession` WHERE `modSession`.`id` = ?  - 
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT DISTINCT `language` FROM `ewal_lexicon_entries` AS `modLexiconEntry` WHERE  ( `modLexiconEntry`.`namespace` = ? AND `modLexiconEntry`.`language` NOT IN (,,,,,,,,,,,,,,,,,,,,,,,,,,,,,) )  
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modActionDom`.`id`, `modActionDom`.`set`, `modActionDom`.`action`, `modActionDom`.`name`, `modActionDom`.`description`, `modActionDom`.`xtype`, `modActionDom`.`container`, `modActionDom`.`rule`, `modActionDom`.`value`, `modActionDom`.`constraint`, `modActionDom`.`constraint_field`, `modActionDom`.`constraint_class`, `modActionDom`.`active`, `modActionDom`.`for_parent`, `modActionDom`.`rank`, `FCSet`.`constraint_class`, `FCSet`.`constraint_field`, `FCSet`.`constraint`, `FCSet`.`template` FROM `ewal_actiondom` AS `modActionDom` JOIN `ewal_fc_sets` `FCSet` ON `modActionDom`.`set` =  `FCSet`.`id` JOIN `ewal_fc_profiles` `Profile` ON FCSet.profile = Profile.id LEFT JOIN `ewal_fc_profiles_usergroups` `ProfileUserGroup` ON Profile.id = ProfileUserGroup.profile LEFT JOIN `ewal_fc_profiles` `UGProfile` ON UGProfile.id = ProfileUserGroup.profile WHERE  ( `modActionDom`.`action` = ? AND `modActionDom`.`for_parent` = ? AND `FCSet`.`active` = ? AND `Profile`.`active` = ? )  ORDER BY modActionDom.rank ASC 
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 263) Error preparing statement for query: SELECT `modActionDom`.`id`, `modActionDom`.`set`, `modActionDom`.`action`, `modActionDom`.`name`, `modActionDom`.`description`, `modActionDom`.`xtype`, `modActionDom`.`container`, `modActionDom`.`rule`, `modActionDom`.`value`, `modActionDom`.`constraint`, `modActionDom`.`constraint_field`, `modActionDom`.`constraint_class`, `modActionDom`.`active`, `modActionDom`.`for_parent`, `modActionDom`.`rank`, `FCSet`.`constraint_class`, `FCSet`.`constraint_field`, `FCSet`.`constraint`, `FCSet`.`template` FROM `ewal_actiondom` AS `modActionDom` JOIN `ewal_fc_sets` `FCSet` ON `modActionDom`.`set` =  `FCSet`.`id` JOIN `ewal_fc_profiles` `Profile` ON FCSet.profile = Profile.id LEFT JOIN `ewal_fc_profiles_usergroups` `ProfileUserGroup` ON Profile.id = ProfileUserGroup.profile LEFT JOIN `ewal_fc_profiles` `UGProfile` ON UGProfile.id = ProfileUserGroup.profile WHERE  ( `modActionDom`.`action` = ? AND `modActionDom`.`for_parent` = ? AND `FCSet`.`active` = ? AND `Profile`.`active` = ? )  ORDER BY modActionDom.rank ASC  - 
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modUser`.`id` AS `modUser_id`, `modUser`.`username` AS `modUser_username`, `modUser`.`password` AS `modUser_password`, `modUser`.`cachepwd` AS `modUser_cachepwd`, `modUser`.`class_key` AS `modUser_class_key`, `modUser`.`active` AS `modUser_active`, `modUser`.`remote_key` AS `modUser_remote_key`, `modUser`.`remote_data` AS `modUser_remote_data`, `modUser`.`hash_class` AS `modUser_hash_class`, `modUser`.`salt` AS `modUser_salt`, `modUser`.`primary_group` AS `modUser_primary_group`, `modUser`.`session_stale` AS `modUser_session_stale`, `modUser`.`sudo` AS `modUser_sudo`, `modUser`.`createdon` AS `modUser_createdon` FROM `ewal_users` AS `modUser` WHERE `modUser`.`username` = ? 
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 263) Error preparing statement for query: SELECT `modUser`.`id` AS `modUser_id`, `modUser`.`username` AS `modUser_username`, `modUser`.`password` AS `modUser_password`, `modUser`.`cachepwd` AS `modUser_cachepwd`, `modUser`.`class_key` AS `modUser_class_key`, `modUser`.`active` AS `modUser_active`, `modUser`.`remote_key` AS `modUser_remote_key`, `modUser`.`remote_data` AS `modUser_remote_data`, `modUser`.`hash_class` AS `modUser_hash_class`, `modUser`.`salt` AS `modUser_salt`, `modUser`.`primary_group` AS `modUser_primary_group`, `modUser`.`session_stale` AS `modUser_session_stale`, `modUser`.`sudo` AS `modUser_sudo`, `modUser`.`createdon` AS `modUser_createdon` FROM `ewal_users` AS `modUser` WHERE `modUser`.`username` = ?  - 
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modUser`.`id` AS `modUser_id`, `modUser`.`username` AS `modUser_username`, `modUser`.`password` AS `modUser_password`, `modUser`.`cachepwd` AS `modUser_cachepwd`, `modUser`.`class_key` AS `modUser_class_key`, `modUser`.`active` AS `modUser_active`, `modUser`.`remote_key` AS `modUser_remote_key`, `modUser`.`remote_data` AS `modUser_remote_data`, `modUser`.`hash_class` AS `modUser_hash_class`, `modUser`.`salt` AS `modUser_salt`, `modUser`.`primary_group` AS `modUser_primary_group`, `modUser`.`session_stale` AS `modUser_session_stale`, `modUser`.`sudo` AS `modUser_sudo`, `modUser`.`createdon` AS `modUser_createdon`, `Profile`.`id` AS `Profile_id`, `Profile`.`internalKey` AS `Profile_internalKey`, `Profile`.`fullname` AS `Profile_fullname`, `Profile`.`email` AS `Profile_email`, `Profile`.`phone` AS `Profile_phone`, `Profile`.`mobilephone` AS `Profile_mobilephone`, `Profile`.`blocked` AS `Profile_blocked`, `Profile`.`blockeduntil` AS `Profile_blockeduntil`, `Profile`.`blockedafter` AS `Profile_blockedafter`, `Profile`.`logincount` AS `Profile_logincount`, `Profile`.`lastlogin` AS `Profile_lastlogin`, `Profile`.`thislogin` AS `Profile_thislogin`, `Profile`.`failedlogincount` AS `Profile_failedlogincount`, `Profile`.`sessionid` AS `Profile_sessionid`, `Profile`.`dob` AS `Profile_dob`, `Profile`.`gender` AS `Profile_gender`, `Profile`.`address` AS `Profile_address`, `Profile`.`country` AS `Profile_country`, `Profile`.`city` AS `Profile_city`, `Profile`.`state` AS `Profile_state`, `Profile`.`zip` AS `Profile_zip`, `Profile`.`fax` AS `Profile_fax`, `Profile`.`photo` AS `Profile_photo`, `Profile`.`comment` AS `Profile_comment`, `Profile`.`website` AS `Profile_website`, `Profile`.`extended` AS `Profile_extended`, `UserSettings`.`user` AS `UserSettings_user`, `UserSettings`.`key` AS `UserSettings_key`, `UserSettings`.`value` AS `UserSettings_value`, `UserSettings`.`xtype` AS `UserSettings_xtype`, `UserSettings`.`namespace` AS `UserSettings_namespace`, `UserSettings`.`area` AS `UserSettings_area`, `UserSettings`.`editedon` AS `UserSettings_editedon` FROM `ewal_users` AS `modUser` LEFT JOIN `ewal_user_attributes` `Profile` ON `modUser`.`id` = `Profile`.`internalKey` LEFT JOIN `ewal_user_settings` `UserSettings` ON `modUser`.`id` = `UserSettings`.`user` WHERE `modUser`.`username` = ? ORDER BY `modUser`.`id` ASC 
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 546) Error  preparing statement: SELECT `modUser`.`id` AS `modUser_id`, `modUser`.`username` AS `modUser_username`, `modUser`.`password` AS `modUser_password`, `modUser`.`cachepwd` AS `modUser_cachepwd`, `modUser`.`class_key` AS `modUser_class_key`, `modUser`.`active` AS `modUser_active`, `modUser`.`remote_key` AS `modUser_remote_key`, `modUser`.`remote_data` AS `modUser_remote_data`, `modUser`.`hash_class` AS `modUser_hash_class`, `modUser`.`salt` AS `modUser_salt`, `modUser`.`primary_group` AS `modUser_primary_group`, `modUser`.`session_stale` AS `modUser_session_stale`, `modUser`.`sudo` AS `modUser_sudo`, `modUser`.`createdon` AS `modUser_createdon`, `Profile`.`id` AS `Profile_id`, `Profile`.`internalKey` AS `Profile_internalKey`, `Profile`.`fullname` AS `Profile_fullname`, `Profile`.`email` AS `Profile_email`, `Profile`.`phone` AS `Profile_phone`, `Profile`.`mobilephone` AS `Profile_mobilephone`, `Profile`.`blocked` AS `Profile_blocked`, `Profile`.`blockeduntil` AS `Profile_blockeduntil`, `Profile`.`blockedafter` AS `Profile_blockedafter`, `Profile`.`logincount` AS `Profile_logincount`, `Profile`.`lastlogin` AS `Profile_lastlogin`, `Profile`.`thislogin` AS `Profile_thislogin`, `Profile`.`failedlogincount` AS `Profile_failedlogincount`, `Profile`.`sessionid` AS `Profile_sessionid`, `Profile`.`dob` AS `Profile_dob`, `Profile`.`gender` AS `Profile_gender`, `Profile`.`address` AS `Profile_address`, `Profile`.`country` AS `Profile_country`, `Profile`.`city` AS `Profile_city`, `Profile`.`state` AS `Profile_state`, `Profile`.`zip` AS `Profile_zip`, `Profile`.`fax` AS `Profile_fax`, `Profile`.`photo` AS `Profile_photo`, `Profile`.`comment` AS `Profile_comment`, `Profile`.`website` AS `Profile_website`, `Profile`.`extended` AS `Profile_extended`, `UserSettings`.`user` AS `UserSettings_user`, `UserSettings`.`key` AS `UserSettings_key`, `UserSettings`.`value` AS `UserSettings_value`, `UserSettings`.`xtype` AS `UserSettings_xtype`, `UserSettings`.`namespace` AS `UserSettings_namespace`, `UserSettings`.`area` AS `UserSettings_area`, `UserSettings`.`editedon` AS `UserSettings_editedon` FROM `ewal_users` AS `modUser` LEFT JOIN `ewal_user_attributes` `Profile` ON `modUser`.`id` = `Profile`.`internalKey` LEFT JOIN `ewal_user_settings` `UserSettings` ON `modUser`.`id` = `UserSettings`.`user` WHERE `modUser`.`username` = ? ORDER BY `modUser`.`id` ASC  - 
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modSession`.`id` AS `modSession_id`, `modSession`.`access` AS `modSession_access`, `modSession`.`data` AS `modSession_data` FROM `ewal_session` AS `modSession` WHERE `modSession`.`id` = ? 
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 263) Error preparing statement for query: SELECT `modSession`.`id` AS `modSession_id`, `modSession`.`access` AS `modSession_access`, `modSession`.`data` AS `modSession_data` FROM `ewal_session` AS `modSession` WHERE `modSession`.`id` = ?  - 
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:39] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modExtensionPackage`.`id`, `modExtensionPackage`.`namespace`, `modExtensionPackage`.`name`, `modExtensionPackage`.`path`, `modExtensionPackage`.`table_prefix`, `modExtensionPackage`.`service_class`, `modExtensionPackage`.`service_name`, `modExtensionPackage`.`created_at`, `modExtensionPackage`.`updated_at`, Namespace.path AS namespace_path FROM `ewal_extension_packages` AS `modExtensionPackage` JOIN `ewal_namespaces` `Namespace` ON `modExtensionPackage`.`namespace` =  `Namespace`.`name` ORDER BY namespace ASC 
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modSession`.`id` AS `modSession_id`, `modSession`.`access` AS `modSession_access`, `modSession`.`data` AS `modSession_data` FROM `ewal_session` AS `modSession` WHERE `modSession`.`id` = ? 
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 263) Error preparing statement for query: SELECT `modSession`.`id` AS `modSession_id`, `modSession`.`access` AS `modSession_access`, `modSession`.`data` AS `modSession_data` FROM `ewal_session` AS `modSession` WHERE `modSession`.`id` = ?  - 
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT DISTINCT `language` FROM `ewal_lexicon_entries` AS `modLexiconEntry` WHERE  ( `modLexiconEntry`.`namespace` = ? AND `modLexiconEntry`.`language` NOT IN (,,,,,,,,,,,,,,,,,,,,,,,,,,,,,) )  
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modActionDom`.`id`, `modActionDom`.`set`, `modActionDom`.`action`, `modActionDom`.`name`, `modActionDom`.`description`, `modActionDom`.`xtype`, `modActionDom`.`container`, `modActionDom`.`rule`, `modActionDom`.`value`, `modActionDom`.`constraint`, `modActionDom`.`constraint_field`, `modActionDom`.`constraint_class`, `modActionDom`.`active`, `modActionDom`.`for_parent`, `modActionDom`.`rank`, `FCSet`.`constraint_class`, `FCSet`.`constraint_field`, `FCSet`.`constraint`, `FCSet`.`template` FROM `ewal_actiondom` AS `modActionDom` JOIN `ewal_fc_sets` `FCSet` ON `modActionDom`.`set` =  `FCSet`.`id` JOIN `ewal_fc_profiles` `Profile` ON FCSet.profile = Profile.id LEFT JOIN `ewal_fc_profiles_usergroups` `ProfileUserGroup` ON Profile.id = ProfileUserGroup.profile LEFT JOIN `ewal_fc_profiles` `UGProfile` ON UGProfile.id = ProfileUserGroup.profile WHERE  ( `modActionDom`.`action` = ? AND `modActionDom`.`for_parent` = ? AND `FCSet`.`active` = ? AND `Profile`.`active` = ? )  ORDER BY modActionDom.rank ASC 
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 263) Error preparing statement for query: SELECT `modActionDom`.`id`, `modActionDom`.`set`, `modActionDom`.`action`, `modActionDom`.`name`, `modActionDom`.`description`, `modActionDom`.`xtype`, `modActionDom`.`container`, `modActionDom`.`rule`, `modActionDom`.`value`, `modActionDom`.`constraint`, `modActionDom`.`constraint_field`, `modActionDom`.`constraint_class`, `modActionDom`.`active`, `modActionDom`.`for_parent`, `modActionDom`.`rank`, `FCSet`.`constraint_class`, `FCSet`.`constraint_field`, `FCSet`.`constraint`, `FCSet`.`template` FROM `ewal_actiondom` AS `modActionDom` JOIN `ewal_fc_sets` `FCSet` ON `modActionDom`.`set` =  `FCSet`.`id` JOIN `ewal_fc_profiles` `Profile` ON FCSet.profile = Profile.id LEFT JOIN `ewal_fc_profiles_usergroups` `ProfileUserGroup` ON Profile.id = ProfileUserGroup.profile LEFT JOIN `ewal_fc_profiles` `UGProfile` ON UGProfile.id = ProfileUserGroup.profile WHERE  ( `modActionDom`.`action` = ? AND `modActionDom`.`for_parent` = ? AND `FCSet`.`active` = ? AND `Profile`.`active` = ? )  ORDER BY modActionDom.rank ASC  - 
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modUser`.`id` AS `modUser_id`, `modUser`.`username` AS `modUser_username`, `modUser`.`password` AS `modUser_password`, `modUser`.`cachepwd` AS `modUser_cachepwd`, `modUser`.`class_key` AS `modUser_class_key`, `modUser`.`active` AS `modUser_active`, `modUser`.`remote_key` AS `modUser_remote_key`, `modUser`.`remote_data` AS `modUser_remote_data`, `modUser`.`hash_class` AS `modUser_hash_class`, `modUser`.`salt` AS `modUser_salt`, `modUser`.`primary_group` AS `modUser_primary_group`, `modUser`.`session_stale` AS `modUser_session_stale`, `modUser`.`sudo` AS `modUser_sudo`, `modUser`.`createdon` AS `modUser_createdon` FROM `ewal_users` AS `modUser` WHERE `modUser`.`username` = ? 
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 263) Error preparing statement for query: SELECT `modUser`.`id` AS `modUser_id`, `modUser`.`username` AS `modUser_username`, `modUser`.`password` AS `modUser_password`, `modUser`.`cachepwd` AS `modUser_cachepwd`, `modUser`.`class_key` AS `modUser_class_key`, `modUser`.`active` AS `modUser_active`, `modUser`.`remote_key` AS `modUser_remote_key`, `modUser`.`remote_data` AS `modUser_remote_data`, `modUser`.`hash_class` AS `modUser_hash_class`, `modUser`.`salt` AS `modUser_salt`, `modUser`.`primary_group` AS `modUser_primary_group`, `modUser`.`session_stale` AS `modUser_session_stale`, `modUser`.`sudo` AS `modUser_sudo`, `modUser`.`createdon` AS `modUser_createdon` FROM `ewal_users` AS `modUser` WHERE `modUser`.`username` = ?  - 
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modUser`.`id` AS `modUser_id`, `modUser`.`username` AS `modUser_username`, `modUser`.`password` AS `modUser_password`, `modUser`.`cachepwd` AS `modUser_cachepwd`, `modUser`.`class_key` AS `modUser_class_key`, `modUser`.`active` AS `modUser_active`, `modUser`.`remote_key` AS `modUser_remote_key`, `modUser`.`remote_data` AS `modUser_remote_data`, `modUser`.`hash_class` AS `modUser_hash_class`, `modUser`.`salt` AS `modUser_salt`, `modUser`.`primary_group` AS `modUser_primary_group`, `modUser`.`session_stale` AS `modUser_session_stale`, `modUser`.`sudo` AS `modUser_sudo`, `modUser`.`createdon` AS `modUser_createdon`, `Profile`.`id` AS `Profile_id`, `Profile`.`internalKey` AS `Profile_internalKey`, `Profile`.`fullname` AS `Profile_fullname`, `Profile`.`email` AS `Profile_email`, `Profile`.`phone` AS `Profile_phone`, `Profile`.`mobilephone` AS `Profile_mobilephone`, `Profile`.`blocked` AS `Profile_blocked`, `Profile`.`blockeduntil` AS `Profile_blockeduntil`, `Profile`.`blockedafter` AS `Profile_blockedafter`, `Profile`.`logincount` AS `Profile_logincount`, `Profile`.`lastlogin` AS `Profile_lastlogin`, `Profile`.`thislogin` AS `Profile_thislogin`, `Profile`.`failedlogincount` AS `Profile_failedlogincount`, `Profile`.`sessionid` AS `Profile_sessionid`, `Profile`.`dob` AS `Profile_dob`, `Profile`.`gender` AS `Profile_gender`, `Profile`.`address` AS `Profile_address`, `Profile`.`country` AS `Profile_country`, `Profile`.`city` AS `Profile_city`, `Profile`.`state` AS `Profile_state`, `Profile`.`zip` AS `Profile_zip`, `Profile`.`fax` AS `Profile_fax`, `Profile`.`photo` AS `Profile_photo`, `Profile`.`comment` AS `Profile_comment`, `Profile`.`website` AS `Profile_website`, `Profile`.`extended` AS `Profile_extended`, `UserSettings`.`user` AS `UserSettings_user`, `UserSettings`.`key` AS `UserSettings_key`, `UserSettings`.`value` AS `UserSettings_value`, `UserSettings`.`xtype` AS `UserSettings_xtype`, `UserSettings`.`namespace` AS `UserSettings_namespace`, `UserSettings`.`area` AS `UserSettings_area`, `UserSettings`.`editedon` AS `UserSettings_editedon` FROM `ewal_users` AS `modUser` LEFT JOIN `ewal_user_attributes` `Profile` ON `modUser`.`id` = `Profile`.`internalKey` LEFT JOIN `ewal_user_settings` `UserSettings` ON `modUser`.`id` = `UserSettings`.`user` WHERE `modUser`.`username` = ? ORDER BY `modUser`.`id` ASC 
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 546) Error  preparing statement: SELECT `modUser`.`id` AS `modUser_id`, `modUser`.`username` AS `modUser_username`, `modUser`.`password` AS `modUser_password`, `modUser`.`cachepwd` AS `modUser_cachepwd`, `modUser`.`class_key` AS `modUser_class_key`, `modUser`.`active` AS `modUser_active`, `modUser`.`remote_key` AS `modUser_remote_key`, `modUser`.`remote_data` AS `modUser_remote_data`, `modUser`.`hash_class` AS `modUser_hash_class`, `modUser`.`salt` AS `modUser_salt`, `modUser`.`primary_group` AS `modUser_primary_group`, `modUser`.`session_stale` AS `modUser_session_stale`, `modUser`.`sudo` AS `modUser_sudo`, `modUser`.`createdon` AS `modUser_createdon`, `Profile`.`id` AS `Profile_id`, `Profile`.`internalKey` AS `Profile_internalKey`, `Profile`.`fullname` AS `Profile_fullname`, `Profile`.`email` AS `Profile_email`, `Profile`.`phone` AS `Profile_phone`, `Profile`.`mobilephone` AS `Profile_mobilephone`, `Profile`.`blocked` AS `Profile_blocked`, `Profile`.`blockeduntil` AS `Profile_blockeduntil`, `Profile`.`blockedafter` AS `Profile_blockedafter`, `Profile`.`logincount` AS `Profile_logincount`, `Profile`.`lastlogin` AS `Profile_lastlogin`, `Profile`.`thislogin` AS `Profile_thislogin`, `Profile`.`failedlogincount` AS `Profile_failedlogincount`, `Profile`.`sessionid` AS `Profile_sessionid`, `Profile`.`dob` AS `Profile_dob`, `Profile`.`gender` AS `Profile_gender`, `Profile`.`address` AS `Profile_address`, `Profile`.`country` AS `Profile_country`, `Profile`.`city` AS `Profile_city`, `Profile`.`state` AS `Profile_state`, `Profile`.`zip` AS `Profile_zip`, `Profile`.`fax` AS `Profile_fax`, `Profile`.`photo` AS `Profile_photo`, `Profile`.`comment` AS `Profile_comment`, `Profile`.`website` AS `Profile_website`, `Profile`.`extended` AS `Profile_extended`, `UserSettings`.`user` AS `UserSettings_user`, `UserSettings`.`key` AS `UserSettings_key`, `UserSettings`.`value` AS `UserSettings_value`, `UserSettings`.`xtype` AS `UserSettings_xtype`, `UserSettings`.`namespace` AS `UserSettings_namespace`, `UserSettings`.`area` AS `UserSettings_area`, `UserSettings`.`editedon` AS `UserSettings_editedon` FROM `ewal_users` AS `modUser` LEFT JOIN `ewal_user_attributes` `Profile` ON `modUser`.`id` = `Profile`.`internalKey` LEFT JOIN `ewal_user_settings` `UserSettings` ON `modUser`.`id` = `UserSettings`.`user` WHERE `modUser`.`username` = ? ORDER BY `modUser`.`id` ASC  - 
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoquery.class.php : 652) Could not construct or prepare query because it is invalid or could not connect: SELECT `modSession`.`id` AS `modSession_id`, `modSession`.`access` AS `modSession_access`, `modSession`.`data` AS `modSession_data` FROM `ewal_session` AS `modSession` WHERE `modSession`.`id` = ? 
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR @ /mnt/sdb1/quo-group/core/xpdo/om/xpdoobject.class.php : 263) Error preparing statement for query: SELECT `modSession`.`id` AS `modSession_id`, `modSession`.`access` AS `modSession_access`, `modSession`.`data` AS `modSession_data` FROM `ewal_session` AS `modSession` WHERE `modSession`.`id` = ?  - 
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 17:38:47] (ERROR in xPDOConnection::connect @ /mnt/sdb1/quo-group/core/xpdo/xpdo.class.php : 3114) SQLSTATE[HY000] [1045] Access denied for user 'laval_root'@'localhost' (using password: YES)
[2018-04-23 18:06:35] (ERROR @ /mnt/sdb1/quo-group/core/model/modx/modcachemanager.class.php : 344) Error caching lexicon topic lexicon/ru/core/resource
[2018-04-23 18:06:45] (ERROR @ /mnt/sdb1/quo-group/core/model/modx/modcachemanager.class.php : 344) Error caching lexicon topic lexicon/ru/core/resource
[2018-04-23 18:17:35] (ERROR @ /mnt/sdb1/quo-group/core/model/modx/modcachemanager.class.php : 394) Error caching action map mgr/actions
[2018-04-23 18:17:35] (ERROR @ /mnt/sdb1/quo-group/core/model/modx/modcachemanager.class.php : 344) Error caching lexicon topic lexicon/ru/core/resource
[2018-04-23 18:33:43] (ERROR @ /mnt/sdb1/quo-group/core/model/modx/modcachemanager.class.php : 344) Error caching lexicon topic lexicon/ru/core/element
[2018-04-23 18:41:50] (ERROR @ /mnt/sdb1/quo-group/core/model/modx/modcachemanager.class.php : 344) Error caching lexicon topic lexicon/ru/core/resource
